FakeServer/Common/CommonHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace FakeServer.Common
+{
+    public static class CommonHelper
+    {
+        /// <summary>
+        /// Checks if input string contains any of the provided substrings.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="substrings"></param>
+        /// <returns></returns>
+        public static bool ContainsAny(this string input, params string[] substrings)
```

Comment:
Always use LINQ if possible. This method can be:

```cs
public static bool ContainsAny(this string input, params string[] substrings) => substrings.Any(e => input.Contains(e));

// Or even

public static bool ContainsAny(this string input, params string[] substrings) => substrings.Any(input.Contains);
```

But actually this whole Helper method can be replaced with a common LINQ query.